chsdi/templates/luftbilder/viewer.mako

```diff
@@ -1,13 +1,10 @@
 <%namespace name="lubis_map" file="lubis_map.mako"/>
 
 <%
-  from pyramid.url import route_url
-  try:
-      from urllib2 import urlopen
-  except ImportError:
-      # Python3 fallback
-      from urllib.request import urlopen
   from json import loads
+  from pyramid.url import route_url
+  # TODO: clean-up when only Python 3.x and no longer 2.x are in use
+  from six.moves.urllib.request import urlopen
```

Comment:
The previous code was easier to clean up after the migration ! It would have just required to remove some lines of code. Now we will need to search for the python3 library that implement `urlopen`, so IMHO the previous code should be left as is, just add the `TODO` comment.

Reply:
IMHO this is even simpler. We only need to remove `six.moves` from `six.moves.urllib.request`, so it will be `from urllib.request import urlopen` aftwards. The TODO comment is already in place.

Reply:
This is true if you know out of the head that in python3 it is `from urllib.request import urlopen` 😉 otherwise not simpler as you have to search for the correct import in python3 first, while the other code is clear from the code what import is fort python2 and what for python3.